tidb.go

```diff
@@ -243,9 +243,24 @@ func NewStore(uri string) (kv.Storage, error) {
 
 var queryStmtTable = []string{"explain", "select", "show", "execute", "describe", "desc"}
 
+func trimSQL(sql string) string {
```

Comment:
If comments start with -- or # the entire line is comment. Only /**/ can mix with sql statement.